Examples/Run/Reconstruction/DD4hep/DD4hepSeedingExample.cpp

```diff
@@ -6,10 +6,26 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+#include "Acts/Geometry/GeometryIdentifier.hpp"
 #include "ActsExamples/DD4hepDetector/DD4hepDetector.hpp"
 
 #include "SeedingExample.hpp"
 
 int main(int argc, char* argv[]) {
-  return runSeedingExample(argc, argv, std::make_shared<DD4hepDetector>());
+  /// selection must not have duplicates.
+  std::vector<Acts::GeometryIdentifier> layersForSeeding = {
+      // open detector barrel layers
+      // the selection intentionally contains duplicates to demonstrate the
+      // automatic selection normalization. setting only the volume already
+      // selects all layers within it. the explicit layers in the selection
+      // should have no effect.
+      Acts::GeometryIdentifier().setVolume(13),
```

Comment:
I'm afraid including all the layers might lead to a lot of duplication.

Reply:
The layers for open data detector are added in the json file as you suggested below.